src/index.js

```diff
@@ -0,0 +1,276 @@
+import _ from 'lodash';
+import './style.css';
+import 'bootstrap/dist/css/bootstrap.min.css'; //eslint-disable-line
+import menu from './assets/menu.jpg';
+import map from './assets/map.png';
+
+function component1() {
+  const element = document.createElement('header');
+  // Lodash, now imported by this script
+  const container_fluid = document.createElement('div'); //eslint-disable-line
+  container_fluid.setAttribute('class','container-fluid w-50'); //eslint-disable-line
+  const row = document.createElement('div');
+  row.setAttribute('class', 'row justify-content-between');
+
+  const about_us = document.createElement('button'); //eslint-disable-line
+  about_us.setAttribute('class','about-us w-25 px-0 py-2 btn-dark'); //eslint-disable-line
+  about_us.innerHTML = _.join(['ABOUT US']); //eslint-disable-line
+  const menu = document.createElement('button');
+  menu.setAttribute('class', 'menu w-25 px-0 py-2 btn-dark');
+  menu.innerHTML = _.join(['MENU']);
+  const contact_us = document.createElement('button'); //eslint-disable-line
+  contact_us.setAttribute('class','contact-us w-25 px-0 py-2 btn-dark'); //eslint-disable-line
+  contact_us.innerHTML = _.join(['CONTACT US']); //eslint-disable-line
+
+  row.append(about_us, menu, contact_us); //eslint-disable-line
+  container_fluid.append(row); //eslint-disable-line
```

Comment:
* [ ] Do not make modfiy linter configs - they represent style guidelines that you share with your team - which is a group of all Microverse students. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/) and [Zendesk](https://microverse.zendesk.com/hc/en-us/articles/1500002085202-February-2nd-2021-linter-errors-as-a-reason-of-project-invalidation-)))